scripts/mysql-migration-test.sh

```diff
@@ -22,6 +22,10 @@ make ARGS="config set SqlSettings.DataSource 'mmuser:mostest@tcp(localhost:3306)
 echo "Setting up fresh db"
 make ARGS="version --config $TMPDIR/config.json" run-cli
 
+echo "Ignoring known MySQL mismatch: ChannelMembers.SchemeGuest"
```

Comment:
can you add these lines to .circleci as well? The 'make test-db-migrations' is not called on CI.

Reply:
Hmm, that’s concerning — we shouldn’t have two sources of truth for this script, given the risk for diversion. I’m assuming it’s a CircleCI anomaly that we can’t just `make test-db-migration`?

Reply:
@lieut-data - yep. This is because of docker usage in the make script. Cannot be done the same way in both command line and CI.